ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_TENANTS_NETWORKS/Tenant_C.yml

```
@@ -84,7 +88,7 @@ tenant_c:
               route_map_out: TEST_OUT
               rcf_in: TEST_RCF_IN()
               rcf_out: TEST_RCF_OUT()
-            nodes: [ DC1-BL2B ]
+            nodes: ['DC1-BL[1|2]B']
```

I guess the quote is necessary around the regex to prevent the inner `]`  to be interpreted maybe we should mention it somewhere

Also I think your regex will match `DC1-BL|B` which you don't want ;) 


```suggestion
            nodes: ['DC1-BL[12]B']
```